vitest.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig, configDefaults } from 'vitest/config'
+import react from '@vitejs/plugin-react'
+
+const SOURCE_FILES = [
+  'app/**',
+  'components/**',
+  'containers/**',
+  'services/**',
+  'libs/**',
+  'utils/**',
+]
+
+export default defineConfig({
+  plugins: [react()],
+  test: {
+    globals: true,
+    setupFiles: './scripts/vitest/setup.ts',
+    environment: 'jsdom',
```

Comment:
I think this would be an issue when we want to test server-side code in the future.

Reply:
If jsdom is not configured in the environment, the setup.ts will fail (which contains mock localstorage, sessionstorage).

If we want test any server side code, we could use [environmentmatchglobs](https://vitest.dev/config/#environmentmatchglobs).

Alternatively, do you have any suggestions～～～?

Reply:
It could work, but I'm not sure how you would deal with the setupFile, I personally don't environment specific stuff in my setup and I'm not really familiar with related configs.

Reply:
Thanks for the suggestion. I made some adjustments and removed environment: jsdom ~~